### PR TITLE
Restructure pomfiles

### PR DESCRIPTION
**Pull request by dizzzz**
mvn tidy:pom
and
	mvn com.github.ekryd.sortpom:sortpom-maven-plugin:sort -Dsort.predefinedSortOrder=custom_1 -Dsort.nrOfIndentSpace=4 -Dsort.sortDependencies=scope,groupId,artifactId -Dsort.keepBlankLines=true

### CODE REVIEW

File: exist-core-jmh/pom.xml [hunk 1]
```diff
@@ -72,10 +73,10 @@
                 <artifactId>maven-shade-plugin</artifactId>
                 <executions>
                     <execution>
-                        <phase>package</phase>
                         <goals>
                             <goal>shade</goal>
                         </goals>
+                        <phase>package</phase>
```
Comment (adamretter) [inline exist-core-jmh/pom.xml]: I think the phase coming after the goals is less readable

File: extensions/exquery/modules/pom.xml [hunk 2]
```diff
@@ -15,15 +15,14 @@
...
-        <developerConnection>scm:git:https://github.com/exist-db/exist.git</developerConnection>
-        <url>scm:git:https://github.com/exist-db/exist.git</url>
-      <tag>HEAD</tag>
-  </scm>
-
     <modules>
         <module>request</module>
     </modules>
 
+    <scm>
```
Comment (adamretter) [inline extensions/exquery/modules/pom.xml]: Why does this come after the modules? surely one of the most important things is knowing where the source code is?

**Comment by dizzzz** [comment]
@adamretter thnx for your feedback; basically I just rand the plugins, assuming that the default settings are well chosen.....

**Comment by dizzzz** [comment]
thnx

**Comment by adamretter** [comment]
@dizzzz you don't need to close it! I think some of the changes are great. The one I think most useful is ordering the dependencies alphabetically

**Comment by joewiz** [comment]
@dizzzz Could you please force push to this branch once more? I think when you closed the PR it might've prevented the Appveyor CI check from completing. I just restarted the 1 failing Travis build.

**Comment by dizzzz** [comment]
I have another idea to improve this PR

**Comment by duncdrum** [comment]
@dizzzz is this still WIP, or have things moved on and we should close ?

**Pull request closed without merging**